torch/csrc/tensor/python_tensor.cpp

```diff
@@ -431,6 +436,9 @@ static bool PyTensorType_Check(PyObject* obj) {
 
 void py_set_default_tensor_type(PyObject* obj) {
   // NOLINTNEXTLINE(cppcoreguidelines-init-variables)
+  TORCH_WARN_ONCE(
```

Comment:
Can you please a test in test_torch.py to make sure this is raised properly similarly to https://github.com/pytorch/pytorch/blob/e7157bd048c663522f4f4101f470d5fd5240b049/test/test_torch.py#L906

Reply:
yes, I'll add the appropriate tests later.